packages/keymanager-server/src/util/lockfile.ts

```diff
@@ -1,24 +1,20 @@
-type Lockfile = {
+export type Lockfile = {
   lockSync(path: string): void;
   unlockSync(path: string): void;
 };
 
-let lockFile: Lockfile | null = null;
+const lockFile: Lockfile = (await import("lockfile")) as Lockfile;
```

Comment:
The point of this file was to import lockfile only when needed. `lockfile` library attaches listeners to the process, so I wanted that to only happen when necessary